### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by ridabensalem**
In this milestone of the Vet-clinic project we have added the following requirements : 
- Create a database named vet_clinic.
- Create a table named animals with the following columns:
- [ ] id: integer
- [ ] name: string
- [ ] date_of_birth: date
- [ ] escape_attempts: integer
- [ ] neutered: boolean
- [ ] weight_kg: decimal
- Copy the SQL into the schema.sql file in the generated repository.
- Insert required  data into the animal's table :
- Copy the SQL into the data.sql file in the generated repository.
- Write queries for the following quests:
- [ ] Find all animals whose name ends in "mon".
- [ ] List the name of all animals born between 2016 and 2019.
- [ ] List the name of all animals that are neutered and have less than 3 escape attempts.
- [ ] List the date of birth of all animals named either "Agumon" or "Pikachu".
- [ ] List name and escape attempts of animals that weigh more than 10.5kg
- [ ] Find all animals that are neutered.
- [ ] Find all animals not named Gabumon.
- [ ] Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
- Take a screenshot of the results of the required queries.
- Copy the SQL  into the queries.sql file in the generated repository.
-  Add the screenshot with the results of your queries to the Pull Request description.

### CODE REVIEW

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @ridabensalem ,

Great job so far! 
However, you are required to add the screenshots of your queries results to the pull request description.

May I highlight the following: 
- [x] Your project looks good and is meeting most of the requirements👏  ✔️ 



## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by ridabensalem** [comment]
Thank you for your feedback but the screenshots were missing only one and I have fixed it they under screenshots title 
image

File: README.md [hunk 1]
```diff
@@ -61,6 +61,15 @@
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 ##  Queries screenshots  <a name="queries"></a>
+<img src="./screenshots/date%20of%20birth.png"/>
+<img src="./screenshots/name%20ending%20with%20mon.png"/>
+<img src="./screenshots/Q3.png"/>
+<img src="./screenshots/Q4.png"/>
+ <img src="https://github.com/ridabensalem/vet-clinic/blob/animals-table/screenshots/Q5%20.png?raw=true"/>
+<img src="./screenshots/Q6.png"/>
+<img src="./screenshots/Q7.png"/>
+<img src="./screenshots/Q8.png"/>
```
Comment (addod19) [inline README.md:71]: ### Optional
- [ ] The queries you have in the PR is not complete, kindly make sure you all the screenshots are added in the PR as requested. The queries in the screenshot below are missing in the PR
Screenshot from 2023-06-05 15-01-15

**Review by addod19 — APPROVED** [review]
### Hi @ridabensalem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job adding some of the queries in the PR


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**